dsl/common/src/main/groovy/net/neoforged/gradle/dsl/common/runs/run/Run.groovy

```diff
@@ -125,6 +125,17 @@ interface Run extends BaseDSLElement<Run>, NamedDSLElement {
     @Optional
     abstract Property<Boolean> getIsServer();
 
+    /**
+     * Indicates if this run is a unit test run.
+     *
+     * @return {@code true} if this run is a unit test run; otherwise, {@code false}.
+     */
+    @Input
+    @DSLProperty
+    @Optional
+    abstract Property<Boolean> getIsUnitTest();
```

Comment:
Please name this `getIsJUnit()` (with a respective propertyName in the annotation) for consistency with the run type.